### DISCUSSION TITLE
OKD User Survey

### DISCUSSION THREAD

**Discussion by JaimeMagiera** [okd-project/okd#865] [Ideas]
At our [August 31st meeting](https://hackmd.io/YJBn04R5TDi5Sm9XbOGwZA#August-31-2021), the OKD Working Group agreed that a brief survey of OKD users could help inform improvements to both the software and the community surrounding it. The survey would be promoted in the various communication channels. 

The group agreed that the survey should consist of no more than a handful of questions. What questions would be the most effective at gaining useful data?

**Comment by DhritiShikhar** [comment]
@JaimeMagiera  I would suggest these questions:

**OKD Software:**

[1] What is the biggest challenge you face using OKD today?

[2] Which critical features do you need that are not available in OKD today?

These two questions would help us understand the pain points users are facing with OKD.

**OKD Documentation:**

[1] How helpful is the official OKD documentation for achieving your goals?

This question will help us identify the struggles users face with documentation.

**OKD Community:**

[1] What could we do to make the OKD community more welcoming?

This question would help us in identifying the steps that could be done to improve the community.

**Reply by JaimeMagiera** [reply]
You've really narrowed in on the important areas. Thanks Dhriti.

**Comment by sandrobonazzola** [comment]
I would ask how useful will be for OKD users having a better integration with KubeVirt for being able to run VMs on OKD.

**Reply by vrutkovs** [reply]
I think we should be more generic - like "which optional operators are you using now" and "which operators are you lacking" (https://github.com/openshift/okd/issues/456). This would help us confirm that existing operators are useful - perhaps some need additional mention - and which operators needs to be prioritized
